scripts/generate-project-outcomes.py

```
@@ -0,0 +1,299 @@
+        i = idx.get(name)
+        return r[i].strip() if (i is not None and i < len(r)) else ''
+
+    out = {}
+    for r in rows:
+        pid = r[C_PID].strip() if len(r) > C_PID else ''
+        if not pid:
+            continue
+        out[pid] = {
+            'selfCheck': col(r, '事業所管部局による点検・改善ー点検結果'),
+            'improvementDirection': col(r, '事業所管部局による点検・改善ー改善の方向性'),
+            'effectAssessment': col(r, '事業所管部局による点検・改善－目標年度における効果測定に関する評価'),
```

Ruff (0.15.21)

[warning] String contains ambiguous `－` (FULLWIDTH HYPHEN-MINUS). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)